rcl/src/rcl/client.c

```diff
@@ -280,7 +281,18 @@ rcl_take_response(
   return RCL_RET_OK;
 }
 
-
+bool rcl_client_is_valid(const rcl_client_t * client)
+{
+  const rcl_client_options_t * options;
+  RCL_CHECK_ARGUMENT_FOR_NULL(
+    client, RCL_RET_INVALID_ARGUMENT, rcl_get_default_allocator());
```

Comment:
Good catch! I believe that would have been the same as `return 1`, which is very not correct.